application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadAutoArchiver.java

```diff
@@ -129,8 +131,12 @@ private void handleArchiveFlow(ThreadChannel threadChannel, MessageEmbed embed)
             }
         };
 
+        HelpThreadsRecord threadRecord =
+                helper.getThreadRecordByChannelId(threadChannel.getIdLong())
+                    .orElseThrow(() -> new NoSuchElementException("No help-thread with such ID"));
```

Comment:
useless extra behavior. just `orElseThrow()` would essentially do the same, with less clutter.